apps/evm/ui/swap/simple/derivedstate-simple-swap-provider.tsx

```diff
@@ -327,11 +355,7 @@ const useSimpleSwapTrade = () => {
   useEffect(() => {
```

Comment:
No reason to keep this in this component, better to put it in the useFallback hook

Reply:
Agreed, was 50/50 on it